src/token.rs

```diff
@@ -29,6 +35,12 @@ impl TryFrom<&str> for TokenType {
     }
 }
 
+impl fmt::Display for Token {
+    fn fmt(&self, f: &mut fmt::Formatter) -> fmt::Result {
+        write!(f, "{}", self.access_token)
+    }
+}
```

Comment:
Is there any reason to implement this instead of using `token.access_token().to_string()`?